RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTGsfTrackVarProducer.cc

```diff
@@ -218,19 +247,9 @@ void EgammaHLTGsfTrackVarProducer::produce(edm::StreamID, edm::Event& iEvent, co
           chi2Value = gsfTracks[trkNr]->normalizedChi2();
         }
 
-        if (std::abs(scAtVtx.dEta()) < dEtaInValue) {
-          // we are allowing them to come from different tracks
-          dEtaInValue = std::abs(scAtVtx.dEta());
-        }
-
-        if (std::abs(scAtVtx.dEta()) < dEtaSeedInValue) {
-          dEtaSeedInValue = std::abs(scAtVtx.dEta() - scRef->position().eta() + scRef->seed()->position().eta());
-        }
-
-        if (std::abs(scAtVtx.dPhi()) < dPhiInValue) {
-          // we are allowing them to come from different tracks
-          dPhiInValue = std::abs(scAtVtx.dPhi());
-        }
+        fillAbsAbleVar(dEtaInValue, scAtVtx.dEta());
+        fillAbsAbleVar(dEtaSeedInValue, scAtVtx.dEta() - scRef->position().eta() + scRef->seed()->position().eta());
```

Comment:
Iiuc, this
```bash
fillAbsAbleVar(dEtaSeedInValue, scAtVtx.dEta() - scRef->position().eta() + scRef->seed()->position().eta());
```

changes the previous call
```bash
if (std::abs(scAtVtx.dEta()) < dEtaSeedInValue) {
  dEtaSeedInValue = std::abs(scAtVtx.dEta() - scRef->position().eta() + scRef->seed()->position().eta());
}
```

to this (assuming `produceAbsValues_ == true`)
```bash
if (std::abs(scAtVtx.dEta() - scRef->position().eta() + scRef->seed()->position().eta()) < std::abs(dEtaSeedInValue)) {
  dEtaSeedInValue = std::abs(scAtVtx.dEta() - scRef->position().eta() + scRef->seed()->position().eta());
}
```

It's not exactly equivalent, so is it effectively equivalent ? Or, is it an intended change ?

Reply:
its a bug fix, sorry should have made that clear.  It was checking dEtaIn was smaller than the current value of dEtaInSeed rather  the existing dEtaInSeed.  The new one is correct. 

Practically it should have zero difference but will result in some minor differences but would not effect physics.